web/beacon-app/src/components/auth/LandingFooter/LandingFooter.tsx

```diff
@@ -19,9 +19,14 @@ function LandingFooter() {
                 <li>
                   <a href={ROUTES.HOME}>Ensign</a>
                 </li>
+                <li>
+                  <a href={EXTERNAL_LINKS.ENSIGN_PRICING} target="_blank" rel="noreferrer">
+                    Pricing
+                  </a>
+                </li>
                 <li>
                   <a href={EXTERNAL_LINKS.DOCUMENTATION} target="_blank" rel="noreferrer">
-                    Documentation
```

Comment:
`Documentation` looked really long in comparison to copy for the other links. Shortening the words makes it more in line with the others and it's common to see `Docs` used on other sites.